pinot-core/src/main/java/com/linkedin/pinot/core/operator/filter/StarTreeIndexOperator.java

```diff
@@ -343,6 +343,11 @@ public void setEndDocId(int endDocId) {
           }
 
           @Override
+          public long getNumEntriesScannedInFilter() {
```

Comment:
why are we returning zero here? Its possible for scan records even when we use star tree. I am ok not implementing this for now and creating another issue to track it